.github/workflows/pr-checks.yml

```
@@ -0,0 +1,103 @@
+on:
+  pull_request:
+    branches: [ main ]
+  repository_dispatch:
+    types: [ ok-to-test ]
+
+jobs:
+  static-analysis:
+    name: Static Analysis
+    uses: ./.github/workflows/_static-analysis.yml
+    with:
+      go_version_file: go.work
```

actionlint (1.7.4)

input "go_version_file" is not defined in "./.github/workflows/_static-analysis.yml" reusable workflow. no input is defined

(workflow-call)